3.0.0/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Gatling is a highly capable load testing tool.
+#
+# Documentation: https://gatling.io/docs/2.3/
```

Comment:
Is now redirected to https://gatling.io/docs/current/

Reply:
Good point! I will put there https://gatling.io/docs/3.0/cheat-sheet/ to make sure it will be still valid even after release of new gatling version. Thanks for pointing that out